src/main/resources/application.properties.EXAMPLE

```diff
@@ -15,11 +15,18 @@ nats.truststore_path=
 nats.key_password=
 nats.store_password=
 
-# messaging configuration
+# new request topics
 metadb.request_filter_topic=
```

Comment:
[requested change]  we can change `metadb` to `smile` as well while we're here making changes